### PR TITLE
Print file db config

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Since the print rows are now being built and formatted in the case processor, there is no need to consume them from PubSub and build the print files row by row, we can instead store the complete rows in the database and extract the whole file with a COPY command once all the rows are present.

The print file service now polls the database print file row table, looking for batches hitting their batch quantity and extracting them into a CSV one they do. This also removes the need for any duplicate checking since the database writes/reads are all strongly transactional. It drastically simplifies the code since it can now be just one, synchronous loop of checking for complete batches, extracting, encrypting and sending them.
# What has changed

Config updated for DB based print files.
# How to test?

Build the images attached to this ticket and make up, run the ATs.
# Links

https://trello.com/c/Lni8psPO/2691-productionise-database-based-print-file-building-spike-8
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[Productionise database based print file building spike (8)](https://trello.com/c/Lni8psPO/2691-productionise-database-based-print-file-building-spike-8)

File: .env [hunk 1]
```diff
@@ -27,6 +28,7 @@ JWT_SECRET=testsecret
 SERVICE_ID=test_service_id
 
 # PUBSUB Emulator
+PUBSUB_PROJECT=project
```
Comment (NickGrantONS) [inline .env:31]: what's this for? The projects should be either `our-project` or `shared-project`

Comment (NickGrantONS) [reply .env:31]: I think this can be removed

**Review by NickGrantONS — APPROVED** [review]
Looks good 👍

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**